kimchi/src/circuits/polynomials/.#foreign_field_common.rs

```diff
@@ -0,0 +1 @@
+volhovm@keshet.3572162:1707819454
```

Comment:
Oh wait!!!

Reply:
Fixed in https://github.com/o1-labs/proof-systems/pull/1962